### DISCUSSION TITLE
Issue with get_data and create_games

### DISCUSSION THREAD

**Discussion by jfacowns** [kyleskom/NBA-Machine-Learning-Sports-Betting#135] [General]
Looks like there might be an issue with the way get_data and create_games are populating the sqlite db. 

For instance, I'll clone the repo and run both scripts to pull in any missing data up until the completion of yesterday's games. I'm looking at games on Jan 9th 2023:

The Lakers are listed as being home to Memphis 
Toronto is listed as being home to New Orleans

Here's the problem though..

Memphis was home to San Antonio on Jan 9 and Toronto didn't even play. New Orleans was in Washington. 

I've had a look at the scripts I mentioned above and don't quite see where things are going off the rails but if the dates and teams are being pulled incorrectly, this obviously has bearing in pulling in the wrong stats, which are used in the model, which would in turn give the wrong projections as far as I understand.

**Comment by nkgilley** [comment]
Thanks for pointing this out.  Looks like it grabbing data from the wrong year.  I'll have a PR for this soon.

**Reply by jfacowns** [reply]
Thanks for the fix guys. Seems like there may be a new issue that has spawned from the fix however. Get_Data no longer seems to update the sqlite DB with the latest games played (games from yesterday for example). Seems like the most recent data the DB has is from March 9 even afte running both Get_Data & Create_Games.

**Reply by nkgilley** [reply]
it seems to be working fine for me. You need Get_Odds_Data too.
